pkg/v22/resource/configmapmigration/create.go

```diff
@@ -67,7 +66,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	if cc.Client.TenantCluster.G8s == nil {
 		r.logger.LogCtx(ctx, "level", "debug", "message", "tenant clients not available")
 		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling reconciliation")
-		reconciliationcanceledcontext.SetCanceled(ctx)
+		resourcecanceledcontext.SetCanceled(ctx)
```

Comment:
This was too cautious. Since the app resource doesn't create the app CR if there are un-migrated settings we are covered.

Reply:
`resourcecanceledcontext.SetCanceled` is needed only in CRUD resource.

Reply:
@kopiczko Thanks TIL :)

Done. PTAL